### PR TITLE
Cldc 1475 shared ownership scheme type

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Add shared ownership scheme type question and page

### CODE REVIEW

File: app/models/form/sales/questions/shared_ownership_type.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Form::Sales::Questions::SharedOwnershipType < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "type"
+    @check_answer_label = "Type of shared ownership sale"
+    @header = "What is the type of shared ownership sale?"
+    @hint_text = "A shared ownership sale is when the purchaser buys up to 75% of the property value and pays rent ro the Private Registered Provider (PRP) on the remaining portion"
```
Comment (baarkerlounger) [inline app/models/form/sales/questions/shared_ownership_type.rb]: Should this say "to the Private..."?

File: app/models/sales_log.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ def validate(record); end
 end
 
 class SalesLog < Log
+  self.inheritance_column = :_type_disabled
```
Comment (baarkerlounger) [inline app/models/sales_log.rb:6]: For anyone wondering without this ActiveRecord assumes a column named type implies Single Table Inheritance applies and removes the column when serialising to JSON.

File: app/models/form/sales/questions/shared_ownership_type.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    @type = "radio"
+    @answer_options = ANSWER_OPTIONS
+    @page = page
+  end
+
+  ANSWER_OPTIONS = {
+    "2" => { "value" => "Shared Ownership" },
```
Comment (baarkerlounger) [inline app/models/form/sales/questions/shared_ownership_type.rb:14]: Do the keys here actually need to be strings?

**Review by baarkerlounger — APPROVED** [review]

File: app/models/form/sales/questions/shared_ownership_type.rb (hunk 3 above)
Comment (kosiakkatrina) [reply app/models/form/sales/questions/shared_ownership_type.rb:14]: That's what we have for the rest of the questions with answer options, so I'd think so

File: app/models/form/sales/questions/shared_ownership_type.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/models/form/sales/questions/shared_ownership_type.rb]: Yup, I just copied it over from the ticket 😅 Good spot!

**Pull request merged**